game/strategy_emptiest_column.py

```diff
@@ -0,0 +1,9 @@
+from game.strategy import Strategy
+
+class StrategyEmptiestColumn(Strategy):
```

Comment:
In these cases you want the class to read nice, so the "Strategy" part should go at the end (like `EmptiestColumnStrategy` or `FirstNonFullColumnStrategy`)